Add vcpkg installation instructions (#663)
tfussell/xlnt: README.md

Documentation for the current release of xlnt is available [here](https://tfussell.gitbooks.io/xlnt/content/).

## Building xlnt - Using vcpkg

> You can download and install xlnt using the [vcpkg](https://github.com/Mixlntosoft/vcpkg) dependency manager:

Reviewer: ```suggestion
You can download and install xlnt using the [vcpkg](https://github.com/microsoft/vcpkg) dependency manager:
```

>     git clone https://github.com/Mixlntosoft/vcpkg.git

Reviewer: ```suggestion
    git clone https://github.com/microsoft/vcpkg.git
```

    cd vcpkg
    ./bootstrap-vcpkg.sh
    ./vcpkg integrate install
    ./vcpkg install xlnt

> The xlnt port in vcpkg is kept up to date by Mixlntosoft team members and community contributors. If the version is out of date, please [xlnteate an issue or pull request](https://github.com/Mixlntosoft/vcpkg) on the vcpkg repository.

Reviewer: ```suggestion
The xlnt port in vcpkg is kept up to date by Microsoft team members and community contributors. If the version is out of date, please [create an issue or pull request](https://github.com/microsoft/vcpkg) on the vcpkg repository.
```
